External/FEXCore/include/FEXCore/IR/IREmitter.h

```diff
@@ -61,6 +61,7 @@ friend class FEXCore::IR::PassManager;
     return _LoadContext(Offset, Class, Size);
   }
   IRPair<IROp_StoreContext> _StoreContext(RegisterClassType Class, uint8_t Size, uint32_t Offset, OrderedNode *ssa0) {
+    assert(Offset != 4048);
```

Comment:
Magic number?

Reply:
Woops, that somehow made it past my self review. That's the value we get when register is invalid (=255), which sometimes happens from the frontend decoder. I'll remove the assert